Update OK/okay guidance in style guide and Vale (#785)
grafana/writers-toolkit: docs/sources/write/style-guide/word-list/index.md

## O

<!-- vale Grafana.OK = NO -->
### OK, okay

> Avoid using _OK_ or _okay_ unless you're referencing the UI, because it's too informal.

Author: ```suggestion
Avoid using _OK_ or _okay_ in technical documentation, because it's too informal. The exceptions are when you're referencing or quoting:
- a user interface
- HTTP status codes or other code
```

## O

<!-- vale Grafana.OK = NO -->

### OK, okay

Avoid using _OK_ or _okay_ in technical documentation because it's too informal. The exceptions are when you're referencing or quoting:

> 
> 

Author: ```suggestion
- A user interface
```

## O

> ### OK, okay

Author: ```suggestion
<!-- vale Grafana.OK = NO -->
### OK, okay
```

> Avoid using _OK_ or _okay_ unless you're referencing the UI, because it's too informal.

Author: ```suggestion
Avoid using _OK_ or _okay_ unless you're referencing the UI, because it's too informal.

<!-- vale Grafana.OK = YES -->
```
